modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -165,6 +165,12 @@
     <string name="widget_no_episode_playing">No episode playing</string>
     <string name="widget_nothing_in_up_next">Nothing in Up Next</string>
     <string name="widget_check_out_discover">Check out Discover for more</string>
+    <string name="widget_preview_episode_title">Episode title</string>
+    <string name="widget_preview_episode_in_up_next_title">Episode in Up Next</string>
+    <string name="widget_preview_time_left_1">25m left</string>
+    <string name="widget_preview_time_left_2">12m left</string>
+    <string name="widget_preview_time_left_3">55m left</string>
+    <string name="widget_preview_time_left_4">1h 36m left</string>
```

Comment:
I think we can remove `widget_preview_time_left_4` since we are not using

Reply:
Good catch! It is used now f501e12fe7bf4f41ca8ecc8134d9f3fa20f4b7c2. I accidentally used `widget_preview_time_left_3` twice.